submission/submitter/src/main/kotlin/ac/uk/ebi/biostd/submission/submitter/SubmissionSubmitter.kt

```diff
@@ -65,16 +65,26 @@ class SubmissionSubmitter(
         try {
             logger.info { "${rqt.accNo} ${rqt.owner} Started processing submission request" }
 
-            rqt.draftKey?.let { draftService.setProcessingStatus(rqt.owner, it) }
+            rqt.draftKey?.let {
+                draftService.setProcessingStatus(rqt.owner, it)
+                logger.info { "${rqt.accNo} ${rqt.owner} Status of draft with key '$it' set to PROCESSING" }
+            }
             val processed = submissionProcessor.processSubmission(rqt)
             parentInfoService.executeCollectionValidators(processed)
-            rqt.draftKey?.let { draftService.setAcceptedStatus(it) }
+            rqt.draftKey?.let {
+                logger.info { "${rqt.accNo} ${rqt.owner} Assigned accNo '${processed.accNo}' to draft with key '$it'" }
```

Comment:
this one I can guess can be moved to `submissionProcessor.processSubmission` where actuallty this happens (I am looking for way to reduce verbosity here )